server/tests/mocks/ingredientsMock.js

```diff
@@ -1,28 +1,72 @@
 class IngredientMock{
 	constructor(){
 		this.ingredientName = "Tomato";
-        this.ingredientAmount = 2;
-        this.ingredientUnit = "whole";
-        this.recipeID = 1;
+	        this.ingredientAmount = 2;
+        	this.ingredientUnit = "whole";
+	        this.recipeID = 1;
+		this.ingredients = [{}, {}, {}]
+		this.ingredients[0].ingredientName = "Tomato";
+		this.ingredients[0].ingredientAmount = 2;
+		this.ingredients[0].ingredientUnit = "whole";
+		this.ingredients[0].recipeID = 1;
+		this.ingredients[0].userID = 3;
+		this.ingredients[0].portion = 2;
+		this.ingredients[1].ingredientName = "Basil";
+		this.ingredients[1].ingredientAmount = 30;
+		this.ingredients[1].ingredientUnit = "g";
+		this.ingredients[1].recipeID = 1;
+		this.ingredients[1].userID = 3;
+		this.ingredients[1].portion = 2;
+		this.ingredients[2].ingredientName = "Mozarella";
+		this.ingredients[2].ingredientAmount = 30;
+		this.ingredients[2].ingredientUnit = "g";
+		this.ingredients[2].recipeID = 1;
+		this.ingredients[2].userID = 3;
+		this.ingredients[2].portion = 2;
+
+
 	}
 
 	sync(){
 
 	}
 
     findAll(arg){
-        if (this.recipeID == arg.where.recipeID) {
+        if (arg.where && this.recipeID == arg.where.recipeID) {
             return [{
                 ingredientName:this.ingredientName, 
                 ingredientAmount:this.ingredientAmount, 
                 ingredientUnit:this.ingredientUnit, 
             }]}
+	else if(arg.include && arg.include[0].where.userID == 3){
```

Comment:
Can we do something like this to make it look cleaner:

function isGreaterThanFive(x) {
     return x > 5;
}

[1, 10, 4, 6].filter(isGreaterThanFive); // Returns [10, 6]